docs/api-reference/extensions/terrain-extension.md

```diff
@@ -9,8 +9,14 @@ To use this extension, first define a terrain source with the prop `operation: '
 
 For each layer that should be fitted to the terrain surface, add the `TerrainExtension` to its `extensions` prop.
 
-<!-- TODO: Codepen demo -->
 <div style={{position:'relative',height:450}}></div>
+<div style={{position:'absolute',transform:'translateY(-450px)',paddingLeft:'inherit',paddingRight:'inherit',left:0,right:0}}>
```

Comment:
Why not use aerial imagery like we do on the https://deck.gl/examples/terrain-layer/? By adjusting the fragment shader we could even make it look more "winter-like" (basically make it black and white)

Reply:
Is there a free aerial imagery dataset? I don't want to include a token in a codepen unless we have to.

Reply:
https://gist.github.com/chrisgervang/9fc4aa7691c633eefaf2887d720f7cd1